Resources/Prototypes/_SpaceCats/Entities/Structures/Machines/Computers/book_printer.yml

```
@@ -0,0 +1,119 @@
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+      - !type:ChangeConstructionNodeBehavior
+        node: machineFrame
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: ItemSlots
+    slots:
+       bookSlot:
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)